Source: me@gmail.com
Upstream: no
Reason: Install supertux executable in /usr/bin rather than /usr/games
Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(版本 5930)
+++ CMakeLists.txt	(工作副本)
@@ -390,7 +390,7 @@
 
 ELSE(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 
-  SET(INSTALL_SUBDIR_BIN "games/")
+  SET(INSTALL_SUBDIR_BIN "bin/")
   SET(INSTALL_SUBDIR_SHARE "share/games/supertux2/")
   SET(INSTALL_SUBDIR_DOC "share/doc/supertux2/")
 
